src/Types.jl

```diff
@@ -0,0 +1,7 @@
+module Types
```

Comment:
I like, putting a types in separate module cleans things up and improves readability

Reply:
I strictly had to do this in this case because otherwise there was an import loop risk